components/content_settings/renderer/brave_content_settings_agent_impl.cc

```diff
@@ -344,6 +346,16 @@ BraveFarblingLevel BraveContentSettingsAgentImpl::GetBraveFarblingLevel() {
     }
   }
 
+  if (!webcompat_features_read_) {
+    GetOrCreateBraveShieldsRemote()->GetWebcompatExceptions(
+        GetOriginOrURL(frame), &webcompat_features_);
+    webcompat_features_read_ = true;
+  }
+  if (std::find(webcompat_features_.begin(), webcompat_features_.end(),
```

Comment:
why not send them as a bitset or something, to simplify the lookup process

Reply:
Good idea -- we're taking a different approach now with content settings.